02-KernelModules_CharacterDevice/Makefile

```
@@ -0,0 +1,9 @@
+obj-m += chrdev.o
+
+BUILD_KERNEL=/lib/modules/$(shell uname -r)/build
+
+all:
+	make -C $(BUILD_KERNEL) M=$(PWD) modules
+
+clean:
+	make -C $(BUILD_KERNEL) M=$(PWD) clean
```

Please add empty line at the end of file
```suggestion
	make -C $(BUILD_KERNEL) M=$(PWD) clean
	
```